aospy/test/data/objects/examples.py

```diff
@@ -39,7 +41,8 @@ def total_precipitation(convection_rain, condensation_rain):
                       'im.landmask.nc')),
     ),
     runs=[example_run],
-    load_grid_data=True
+    load_grid_data=True,
+    grid_attrs={LAND_MASK_STR: 'custom_land_mask', LON_STR: 'custom_lon'}
```

Comment:
Nice, I think this is a good idea to have modified the example data directly to ensure that this works and be able to show this functionality in the tutorial.